agent/util-scripts/pbench-collect-sysinfo

```diff
@@ -157,7 +168,9 @@ done
 if [ $lcl_cnt -gt 0 ]; then
 	# We found local tools, or a label, specified in the tools group
 	# file, so we should collect sysinfo data locally
-        pbench-sysinfo-dump "$sysinfo_path" $lcl_label &
+        if [[ -n "$collect_sysinfo" ]] && [[ "$collect_sysinfo" != "none" ]]; then
```

Comment:
Again, I don't think we want to test for the behavior options here, but right after the location we gather the command line arguments.